api/controllers/pr/index.js

```diff
@@ -23,32 +23,57 @@ exports.index = (req, res) => {
   Promise.all([
     findGithubPrs(github, username),
     findGitlabPrs(gitlab, username),
-    github.users.getByUsername({ username }).then(logCallsRemaining),
+    searchGithubUser(github, username),
+    searchGitlabUser(gitlab, username),
   ])
-    .then(([prs, mrs, user]) => {
-      if (user.data.type !== 'User') {
+    .then(([prs, mrs, user, gitlab_user]) => {
+      let isGitHubUser = true;
+      if (user == null) isGitHubUser = false;
+      let isGitLabUser = true;
+      if (JSON.stringify(gitlab_user) == JSON.stringify([]))
```

Comment:
How come we comparing json here and not doing the same null check as above?

Reply:
gitlab returns [] when no user is found
github returns null when no user is found

Reply:
Can we normalise upstream? Even if just like `return user_data === null ? [] : user_data` or something?

Reply:
We can. I can make this change for you.